client/src/AddFriend.js

```diff
@@ -93,8 +108,11 @@ class AddFriend extends React.Component {
         <h5>Hi ***{this.state.userId}*** {this.state.firstName} {this.state.lastName}! Add your friends below.</h5>
 
         <ul>
-          {this.state.friendships.map(item => (
-            <li key={item}>{item}</li>
+          {this.state.friendList.map((item) => (
+            <li key={item.friendId}>
+              <span>{item.firstName} {item.lastName} </span>
+              <button type='submit' onClick={() => { this.handleRemove(item.friendId) }}>X</button>
```

Comment:
I think a confirmation modal on friend delete would be ideal. It's probably quite rare for someone to choose to delete their friend unless they misspelled the email or something.

In the future, we can have choose to have deleted friends show up in the Suggested panel along with friends that added you first.